### ISSUE TITLE
Getting notifications for all Google Inbox emails, They're not filtered on Mac

### ISSUE THREAD

**Issue by tgrrr** [wavebox/waveboxapp#443]
Hi, I'm getting notifications as each mail comes in with Google Inbox

The goal is to have notifications if an email is unbundled anywhere.

Cheers,

Phil

General Settings:
![wavebox___phil_tgrrr_com___inbox_ _phil_tgrrr_com](https://user-images.githubusercontent.com/4689707/32204377-f134c21a-be3c-11e7-8b96-d69d2b1298f4.png)

Account Settings:
![wavebox___phil_tgrrr_com___inbox_ _phil_tgrrr_com](https://user-images.githubusercontent.com/4689707/32204344-b24e54b2-be3c-11e7-8f68-4d8624ad24dc.png)

[This isn't the same issue as:](https://github.com/wavebox/waveboxapp/issues/269)

* Which version of Wavebox are you using? [e.g. 3.0.0]

3.4

* Which Operating System are you using? [e.g. Windows 10, Mac OSX 10, Ubuntu 14.04]

Mac OSX 10.12.6

**Comment by Thomas101** [comment]
Hi, at the moment Inbox doesn't have support for searching for unbundled messages directly [(info)](https://productforums.google.com/forum/#!topic/inbox/2pO128A3Apo). However, you can emulate the behaviour by excluding labels that are bundled. For example using the following gets you pretty close...

```label:inbox label:unread -has:userlabels -category:promotions -category:forums -category:social```

...depending on how you've got your inbox configured, you may have to change it up a bit, by excluding the labels you've selected to bundle individually. E.g...

```label:inbox label:unread -label:mylabel1 -label:mylabel2 -label:mylabel3 -category:promotions -category:forums -category:social```

...the easiest way to test this is by pasting your search into the inbox search bar. Once you're happy with it add it into the Custom unread Query setting and see how it works.

As for the `Custom unread watch labels` I suspect either `INBOX, UNREAD` or `INBOX` would serve okay there.

**Comment by tgrrr** [comment]
Thanks @Thomas101 ,

The search query 

`label:inbox label:unread`

gives me exactly the messages that I'd like **notifications** for. However, setting this up as above (see screenshots) gives me notifications for all new bundled mail that arrives. 

That is, I only want to be interrupted/notified by emails that I deem important. I understand that the section is for 'unread and sync'. But if there are no custom notification settings, then the notifications should follow the same business rules as above.

Otherwise: I get a notification every time some fool decides to send me spam/bacon.

In other words, my user story is: As a user of Google Inbox and mac, I only want to be notified of mail that I label as important, so that I can continue to focus on the task at hand without being interrupted by notifications for newsletters, nigerian princes, bank notifications, etc.

Thanks.

**Comment by Thomas101** [comment]
`label:inbox label:unread` really should be giving you everything in the inbox bundled or not. For example, I've got a `Wavebox Github` bundle that Github emails fall into. Running `label:inbox label:unread` brings these up in the search.

Which bundles are you trying to exclude? Are they custom folders or ones that inbox comes built in with? If you open up your Google Inbox settings and click the Labels tab what do you have setup there?

![screen shot 2017-11-01 at 12 01 24](https://user-images.githubusercontent.com/103586/32273953-8cb928d8-befc-11e7-8027-3768bc9e824f.png)

**Comment by tgrrr** [comment]
Thanks @Thomas101, that worked.

I found my sweet spot by adding `is:pinned`

`label:inbox label:unread -has:userlabels -category:promotions -category:forums -category:social -category:Updates  -category:Finance -category:promos OR is:pinned`

One suggestion: Is there a way to label the options: 'notification labels/categories etc' for clarity?

**Comment by Thomas101** [comment]
Awesome, glad you got it working. The next beta has some more info on how to get these setup with a link to the knowledge base article etc.
